synapse/storage/data_stores/main/room.py

```diff
@@ -388,7 +444,17 @@ def _get_rooms_paginate_txn(txn):
                         "room_id": room[0],
                         "name": room[1],
                         "canonical_alias": room[2],
-                        "joined_members": room[3],
+                        "joined_members": room[11],
+                        "joined_local_members": room[13],
+                        "version": room[8],
+                        "creator": room[10],
+                        "encryption": room[3],
+                        "federatable": room[5],
+                        "public": room[9],
+                        "join_rules": room[7],
+                        "guest_access": room[4],
+                        "history_visibility": room[6],
+                        "state_events": room[12],
```

Comment:
Any method for the ordering of these? Seems prone to error!

Reply:
What would you like? Alphabetically? I can change the order in sql statement, alternatively.